### ISSUE TITLE
Watchlist not importing to library

### ISSUE THREAD

**Issue by krijos** [123Venom/plugin.video.venom#80]
When trying to import trakt watchlist for tv shows I get the message "Title already exists in the library." or similar to that, despite the title not being imported.

Tried clearing all cache, deleting every cache I could find, and removing/rebuilding library.db. No luck.

My setup utilizes a path substitution from Venom's default TV/Movie paths to a remote server as I found Venom had issues dealing with the dav protocol directly.

Having searched the library.db file the shows do not exist in the file.

The solution is run on an Nvidia Shield.

Feel free to ask about any further info you believe might help find the cause. If you feel comfortable that it's not a code issue I'll keep troubleshooting it on my end and hopefully reach a solution. :)

**Comment by 123Venom** [comment]
Extremely busy week so I may not be able to dive into this till the weekend.  That library module is a bugger and tracking issues in it is no small and quick task so I'll need a good deal of time set aside to find out what's going on.  In the mean time if you could screen shot your library.db.  There's 3 tables in there take one of each.  Also in the mean time check the paths where your TVShows are kept and verify if any title from your Trakt watchlist exists there.

**Comment by krijos** [comment]
Completely understandable, I'm just grateful you're offering your spare time supporting this community, so absolutely no pressure! 😁
Just delievered a new release to a client myself, so I got time to dig into it a little if you have some pointers as to where would be a good part of the code to dig into? I often write small patches to things to grease the gears a little, might be time I actually start sharing them… 😅

I've verified that no watchlist items have been added to the path, however, existing items do keep updating as normal, indicating there's no pathing error.
Included a gdrive share to my library.db so you can query it as you please!
https://drive.google.com/file/d/1tpY8Q0WUYsFGD7yxF7sHEjbbP0khtbK5/view?usp=sharing

**Comment by 123Venom** [comment]
I had a few minutes and curiosity got the better of me so I peaked.  I think I found it since I was able to duplicate the issue.  Mine checked 1-2 titles then seemed to exist when I know I have new titles in my watchlist since the last time I did an import of them.  Turn on the notifications for library service and general notifications in settings.  I have a bad indent issue and when notifications are off the counter for the imports doesn't execute so it terminates.  It will be fixed in next release but in the mean time test turning those on...worked here to get around it but I fixed the dent issue.
